wd_helper.au3

```diff
@@ -961,7 +980,20 @@ Func _WD_LoadWait($sSession, $iDelay = Default, $iTimeout = Default, $sElement =
 		Else
 			$sReadyState = _WD_ExecuteScript($sSession, 'return document.readyState', '', Default, $_WD_JSON_Value)
 			$iErr = @error
-			If $iErr Or $sReadyState = 'complete' Then
+			If $iErr Then
+				$iErr = $_WD_ERROR_Exception
+				$sReadyState = ''
+			Else
+				For $i = 0 To $_WD_READYSTATE__COUNTER - 1
+					If $sReadyState = $aWD_READYSTATE[$i][0] Then
+						$iExt = $i
+						$sReadyState &= ' (' & $aWD_READYSTATE[$i][1] & ')'
+						ExitLoop
+					EndIf
+				Next
+			EndIf
```

Comment:
Did you consider using `_ArraySearch` here?

Reply:
Please make do the trick, as I go sleep to bed in a moment.

Reply:
@mlipok Please take a look at https://github.com/Danp2/au3WebDriver/pull/384/commits/9a969783f9e116f7070f4dc929fb3c80b74568cb and provide your feedback.

Reply:
Is ok